pep-0646.rst

```diff
@@ -680,6 +680,30 @@ between old unannotated code and new versions of libraries which do use
 these type annotations.
 
 
+Alternatives
+============
+
+It should be noted that the approach outlined in this PEP to solve the
+issue of shape checking in numerical libraries is *not* the only approach
+possible. Examples of lighter-weight alternatives include
```

Comment:
We've already mentioned the alternative approach of manually writing `@overload`s for types like `Tensor1D`, `Tensor2D`, etc., along with its disadvantages. Not sure if it needs to be repeated here.

Reply:
True; though a) I want to highlight the difference between runtime analysis (which is a very different kind of beast - wouldn't require lots of different `Tensor` classes or type stubs) and static analysis, b) I want to be more concrete and reference examples of specific work in the same domain, and c) I want to build a bit of gusto for what this could lead to longer-term :) What do you think?

Reply:
Given the SC review, anxious to get this wrapped up, so assuming this is fine - but if you feel uneasy about it, happy to do another PR.